File: CS/WebApplication1/Service1.asmx.cs
using System;
using System.Web;
using System.Web.Services;
using System.Web.Services.Protocols;

namespace WebApplication1 {
	[WebService(Namespace = "http://tempuri.org/")]
	[WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
	public class Service : System.Web.Services.WebService {
		public Service() {
			//Uncomment the following line if using designed components 
			//InitializeComponent(); 
		}

		[WebMethod]
		public string HelloWorld() {
			return "Hello World";
		}
		[WebMethod]
		public ClassLibrary1.DataSet1 FillDS() {
			ClassLibrary1.DataSet1 ds = new ClassLibrary1.DataSet1();
			ds.Tables[0].Rows.Add(new object[] { "AAA", "BBB" });
			ds.Tables[0].Rows.Add(new object[] { "CCC", "DDD" });
			ds.Tables[0].Rows.Add(new object[] { "EEE", "FFF" });
			return ds;
		}

	}
}

File: VB/WebApplication1/Service1.asmx.vb
Imports System
Imports System.Web
Imports System.Web.Services
Imports System.Web.Services.Protocols

Namespace WebApplication1
	<WebService([Namespace] := "http://tempuri.org/")>
	<WebServiceBinding(ConformsTo := WsiProfiles.BasicProfile1_1)>
	Public Class Service
		Inherits System.Web.Services.WebService

		Public Sub New()
			'Uncomment the following line if using designed components 
			'InitializeComponent(); 
		End Sub

		<WebMethod>
		Public Function HelloWorld() As String
			Return "Hello World"
		End Function
		<WebMethod>
		Public Function FillDS() As ClassLibrary1.DataSet1
			Dim ds As New ClassLibrary1.DataSet1()
			ds.Tables(0).Rows.Add(New Object() { "AAA", "BBB" })
			ds.Tables(0).Rows.Add(New Object() { "CCC", "DDD" })
			ds.Tables(0).Rows.Add(New Object() { "EEE", "FFF" })
			Return ds
		End Function

	End Class
End Namespace